.eslintcache~main

```diff
@@ -0,0 +1 @@
t}) {\n\n  let [min, max] = [1, 6];\n  const shakeDace = 30;\n\n  const [dice1, setDice1] = React.useState();\n  const [dice2, setDice2] = React.useState();\n  const [counter, setCounter] = React.useState(0);\n\n  const rolldice = () => {\n    setDice1(Math.floor(Math.random() * ((max - min) + 1) + 1));\n    setDice2(Math.floor(Math.random() * ((max - min) + 1) + 1));\n  };\n\n  useEffect(() => {\n    let timer = setInterval(() => {\n      if(counter >= shakeDace) {\n        setCounter(0);\n      } else {\n        setCounter(counter + 1);\n        rolldice();\n      }\n    }, counter < shakeDace ? 150 : 5000);\n    return () => {\n      clearInterval(timer);\n    };\n  }, [counter]);\n\n  return (\n    <div className=\"dice_wrapper\">\n      <div className=\"dice_middle_block\">\n        <div className=\"dices\">\n          {(dice1 > 0 && dice1 <=6) && <div className={'dice dice-' + dice1}></div>}\n          {(dice2 > 0 && dice2 <=6) && <div className={'dice dice-' + dice2}></div>}\n        </div>\n        {counter === 30 && <div className=\"user-feedback\">What am I feeling?</div>}\n        {counter !== 30 && <div className=\"user-feedback\">Rolling...</div>}\n      </div>\n    </div>\n  );\n\n}\n\nexport default Dice;\n",{"ruleId":"45","severity":1,"message":"46","line":1,"column":8,"nodeType":"47","messageId":"48","endLine":1,"endColumn":12},{"ruleId":"49","replacedBy":"50"},{"ruleId":"51","replacedBy":"52"},{"ruleId":"53","severity":1,"message":"54","line":28,"column":6,"nodeType":"55","endLine":28,"endColumn":12,"suggestions":"56"},{"ruleId":"45","severity":1,"message":"57","line":2,"column":17,"nodeType":"47","messageId":"48","endLine":2,"endColumn":26},{"ruleId":"58","severity":1,"message":"59","line":8,"column":7,"nodeType":"60","endLine":8,"endColumn":35},{"ruleId":"53","severity":1,"message":"61","line":30,"column":6,"nodeType":"55","endLine":30,"endColumn":15,"suggestions":"62"},"no-unused-vars","'logo' is defined but never used.","Identifier","unusedVar","no-native-reassign",["63"],"no-negated-in-lhs",["64"],"react-hooks/exhaustive-deps","React Hook useEffect has missing dependencies: 'config.bites', 'config.light', 'config.things', and 'config.time'. Either include them or remove the dependency array.","ArrayExpression",["65"],"'useEffect' is defined but never used.","jsx-a11y/alt-text","img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.","JSXOpeningElement","React Hook useEffect has a missing dependency: 'rolldice'. Either include it or remove the dependency array.",["66"],"no-global-assign","no-unsafe-negation",{"desc":"67","fix":"68"},{"desc":"69","fix":"70"},"Update the dependencies array to be: [config.bites, config.light, config.things, config.time, logs]",{"range":"71","text":"72"},"Update the dependencies array to be: [counter, rolldice]",{"range":"73","text":"74"},[907,913],"[config.bites, config.light, config.things, config.time, logs]",[738,747],"[counter, rolldice]"]
```

Comment:
hi, man, I know this is not supposed to be here at all. it is in gitignore but nothing works so far. I'll check it later.